src/main/java/step3/RacingGame.java

```diff
@@ -0,0 +1,30 @@
+package step3;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGame {
+    private List<Car> cars = new ArrayList<>();
+
+    public void readyCar(int CarCount) {
```

Comment:
```suggestion
    public void readyCar(int carCount) {
```
일관성있게 카멜케이스로 사용하는 것이 좋을 것 같네요

Reply:
넵 신경써서 작업해보겠습니다!